events/membership.js

```diff
@@ -10,70 +10,75 @@ import User from '../models/user';
 // 'user' and 'secretary' are User objects
 // 'user' = person the email should go to
 // 'secretary' = current SSE secretary
-const sendCongratsEmail = (user, secretary = null) => mailer.sendMail({
```

Comment:
I'm curious why all the formatting changed in these files. Do you know why?

Reply:
That's totally my fault; I haven't used vscode for a hot minute and forgot I had Prettier enabled with format on save (and that the vscode prettier plugin adds its own default prettier settings). Prettier went a... little overzealous.